src/SmartTransactionsController.ts

```diff
@@ -330,22 +369,27 @@ export default class SmartTransactionsController extends BaseController<
     });
   }
 
-  async updateSmartTransactions() {
+  async updateSmartTransactions(
+    networkClientId?: NetworkClientId,
```

Comment:
Nit: What are your thoughts on placing this in an (optional) options object rather than adding this as a positional argument? I feel like if we ever need to update the signature of this method for any reason then we could do that easily without having to displace this argument. (Same goes for the other cases in this PR where you've added optional arguments.) If this is how you've been doing it in other PRs and are doing it this way for consistency, that's fine, but I just wanted to pose this question in case it was flexible.

Reply:
That works for me!

Reply:
> If this is how you've been doing it in other PRs and are doing it this way for consistency, that's fine, but I just wanted to pose this question in case it was flexible.

The pattern hasn't been super consistent across the refactor up to this point. However I think you're correct that an optional options object is the best approach here so I think it's reasonable to adopt that pattern going forward.